Sources/eppo/EppoClient.swift

```diff
@@ -248,6 +248,11 @@ public class EppoClient {
         }
     }
 
+    /// Returns the current configuration stored in the client.
+    public func getFlagsConfiguration() -> Configuration? {
+        return self.configurationStore.getConfiguration()
```

Comment:
This method already returns the `Configuration` object that could be used directly for offline initialization

https://github.com/Eppo-exp/eppo-ios-sdk/blob/20acd299dda1a63fddfdeebea6305d96bead6571/Sources/eppo/ConfigurationStore.swift#L64